modules/core/src/effects/lighting/lighting-effect.ts

```diff
@@ -13,7 +13,7 @@ import shadow from '../../shaderlib/shadow/shadow';
 
 import type Layer from '../../lib/layer';
 import type {Effect, EffectContext, PreRenderOptions} from '../../lib/effect';
-import {LightingProps} from '@luma.gl/shadertools';
+import type Viewport from '../../viewports/viewport';
```

Comment:
Nit: I have remove all default exports inside other frameworks... just a suggestion to try to avoid them in new deck code.